gb-apu/src/apu.rs

```diff
@@ -66,7 +66,9 @@ impl Apu {
             .with_max_sample_rate();
         let err_fn = |err| eprintln!("an error occurred on the output audio stream: {}", err);
         let sample_format = supported_config.sample_format();
-        let config: StreamConfig = supported_config.into();
+        let mut config: StreamConfig = supported_config.into();
+        config.buffer_size =
```

Comment:
I don't know what this does but it breaks compatibility with Windows (#689)
Otherwise works outstandingly well

Reply:
This will be addressed in an other branch